### ISSUE TITLE
i18n support

### ISSUE THREAD

**Issue by ccorda** [patronage/bubs-next#58]
Occasionally we have sites that need multilingual support.

Can we develop a recommendation (docs at least, maybe with some code support) for how to handle on both the WP and Next side of things?

Some quick notes of research here:

* https://nextjs.org/docs/advanced-features/i18n-routing
* https://wpml.org/
* https://wpml.org/forums/topic/does-wpml-have-support-for-graphql/
* https://github.com/rburgst/wp-graphql-wpml

**Comment by nickwebdesign** [comment]
Saw this issue pop up in Slack. Thought I'd add a few notes to it based on my experience with WPML. I didn't love using WPML, although after working with it a bit I think Stephen and I both understand the quirks of it enough so that using it again would be a little smoother. 

Kyle, WPML creates a new post ID for each translated post so in theory,  I think the logic would be something like:
```
IF lang IS eng {
       load POST_ID = 1234
}  ELSEIF lang = es {
	load POST_ID = 5678
}
```

The benefit of WPML is that all posts across languages are contained in one WP admin interface. This means posts can be duplicated and acf field copied over making it a little easier for WordPress admins. 

I personally prefer the multisite approach we've used for Informed Immigrant and some other sites. Is the current bubs-next set up able to work with multisite in its current state or is that something we'd have to test and build out. 

Found this plugin as well, may be worth comparing to wpml although it seems slightly less widely used https://wordpress.org/plugins/polylang/ - looks like there's some graphql support for it: https://github.com/valu-digital/wp-graphql-polylang
